### PR TITLE
26 implement tests to validate process monitoring

### PR DESCRIPTION
**Pull request by AssahBismarkabah**
added unit test for validating the process monitoring using the script to track and report the running processes

### CODE REVIEW

**Comment by AssahBismarkabah** [comment]
hello @stephane-segning

**Comment by AssahBismarkabah** [comment]
hello sir i cant view your reviews sir i dont know why

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Ok, review this please

File: services/tests/process_tracking.py
[the remark this answers is not in the document]
Comment (AssahBismarkabah) [reply services/tests/process_tracking.py:1]: hello sir it is fine now

[the remark this answers is not in the document]
Comment (AssahBismarkabah) [reply services/tests/process_tracking.py:1]: so sir as i was saying the process_tracking.py is the script involves in tracking the running processes while the test_process_tracking.py validates the running processing using by importing the function from the track script

[the remark this answers is not in the document]
Comment (AssahBismarkabah) [reply services/tests/process_tracking.py:1]: Screenshot from 2023-11-10 16-59-09

**Review by stephane-segning — APPROVED** [review]
Good work @AssahBismark ! Now you can merge!

**Comment by AssahBismarkabah** [comment]
sir the merge is still blocked

**Pull request merged**